backend/app/models/content.py

```diff
@@ -1,20 +1,39 @@
 """Content Model"""
 
 from datetime import datetime
+from uuid import uuid4
 from marshmallow_sqlalchemy import SQLAlchemyAutoSchema
+from marshmallow import fields
 from ..extensions import DB as db
+from ..models.comment import Comment, CommentSchema
 
 
 class Content(db.Model):
     """Content Model"""
 
+    __tablename__ = "content"
+    __table_args__ = {"extend_existing": True}
+
     id = db.Column(db.String(100), primary_key=True)
     title = db.Column(db.String(100), nullable=False)
-    content = db.Column(db.String(5000), nullable=False)
+    body = db.Column(db.String(5000), nullable=False)
     created_at = db.Column(db.DateTime, default=datetime.now(), nullable=False)
     updated_at = db.Column(db.DateTime, default=datetime.now(), nullable=False)
     deleted_at = db.Column(db.DateTime, nullable=True)
 
+    # Add relationship to comments
+    comments = db.relationship(
+        "Comment",
```

Comment:
I think it's also good to define the inverse relationship on the Comment model

- [ ] Check and define inverse relationship